src/model/UserModel.ts

```diff
@@ -44,6 +55,42 @@ async function insertUser(email: string, password: string, fullName: string) {
   );
 }
 
+async function updateUser(
+  id: number,
+  profile: {
```

Comment:
Why not just use the `UserProfile` type you created? Why do you write them again?